app/views/spaces/show.html.erb

```diff
@@ -1,2 +1,15 @@
 <h1>Spaces#show</h1>
 <p>Find me in app/views/spaces/show.html.erb</p>
+
+<ul class="list-group">
+  <% spaces.each do |space| %>
```

Comment:
You would access the individual space through @space as defined in the controller. There is no need for an iterator (.each) here.